### PR TITLE
Expand explanation of available branches

### PR DESCRIPTION
**Pull request by gfscott**
I think it would be good to provide more context for the relationship between the different branches and the steps in the tutorial!

### CODE REVIEW

**Comment by markiiikram** [comment]
Thanks @katiedavis. I'll update for all the other branches too.

**Comment by katiedavis** [comment]
> Thanks @katiedavis. I'll update for all the other branches too.

Thanks!

File: README.md [hunk 1]
```diff
@@ -2,21 +2,30 @@
 
 This repository contains the completed app and starter files for the [Build a Shopify app with Node and React](#) tutorial.
 
-The completed app is the master branch, and each starter file has its own branch. For example, To start the tutorial at [Build your user interface with Polaris](#) check out and use `build-your-user-interface-with-polaris-starter-files`.
+## Available branches for each step in the tutorial
+
+Each step of the [Build a Shopify app with Node and React](https://developers.shopify.com/tutorials/build-a-shopify-app-with-node-and-react) tutorial has a corresponding branch in this repository. Checkout any of these branches to view the demo project at that stage, or to use it as the starting point for your own app. For example, To start the tutorial at [Build your user interface with Polaris](#) check out and use `build-your-user-interface-with-polaris-starter-files`.
```
Comment (markiiikram) [inline README.md]: looks like (https://developers.shopify.com/tutorials/build-a-shopify-app-with-node-and-react/build-your-user-interface-with-polaris) is missing for (#)

File: README.md [hunk 2]
```diff
@@ -2,21 +2,30 @@
 
 This repository contains the completed app and starter files for the [Build a Shopify app with Node and React](#) tutorial.
```
Comment (markiiikram) [inline README.md]: here too (#) (https://developers.shopify.com/tutorials/build-a-shopify-app-with-node-and-react)

**Comment by markiiikram** [comment]
@gfscott left comments as this is your PR, but happy to fix if you want

File: README.md (hunk 1 above)
Comment (markiiikram) [reply README.md]: as well as `build-your-user-interface-with-polaris-starter-files`

File: README.md [hunk 3]
```diff
@@ -2,21 +2,30 @@
...
+6. Completed App: [`master`](https://github.com/Shopify/shopify-demo-app-node-react/tree/master)
 
 ## Using starter files
 
 You can use these starter files to skip ahead and learn a certain section or fix an app in a broken state.
 
 If you plan to skip ahead, then make sure that you first complete these setup instructions:
 
-1. [Install the latest stable version of Node.js.](#)
+1. [Install the latest stable version of Node.js.]('https://developers.shopify.com/tutorials/build-a-shopify-app-with-node-and-react/set-up-your-app#install-the-latest-stable-version')
```
Comment (markiiikram) [inline README.md]: @gfscott all of these should be without single quotes

**Pull request merged**